packages/create-cloudflare/templates-experimental/next/templates/wrangler.jsonc

```diff
@@ -5,7 +5,6 @@
   "compatibility_flags": [
     "nodejs_compat"
   ],
-  "minify": true,
```

Comment:
why removing this? (is true the default)?

Reply:
The default is `false`.
The PR removes this to be consistent with other template and to help with inspecting the code.
I don't think there is any cons?

Reply:
I forgot to mention that the code is minified by OpenNext unless the env var `OPEN_NEXT_DEBUG` is set to 1